GirafAPI/Endpoints/LoginEndpoints.cs

```diff
@@ -33,7 +33,7 @@ public static void MapLoginEndpoint(this WebApplication app)
                 var claims = new List<Claim>
                 {
                     new Claim(ClaimTypes.NameIdentifier, user.Id),
-                    new Claim(ClaimTypes.Name, user.UserName)
+                    new Claim(ClaimTypes.Name, user.UserName!)
```

Comment:
This seems like a pretty bad solution guys, why would hack the compiler and potentially make it harder to debug?

Reply:
I dont understand

Reply:
My point is simply that ! is generally a bad practice, in my opinion its better to handle these errors gracefully. Even if you are sure that UserName is ensured to be present at all times a better solution would be a fallback or making the field required.